### ISSUE TITLE
Full CRUD on all items

### ISSUE THREAD

**Issue by noahkoch** [opennebraska/api-localboards-org#5]
I'm assuming we could leave out states and counties from that list. Cities we could also probably seed the DB with.

**Comment by trakaus** [comment]
So we would need methods for the following:
1) Cities
2) Boards
3) Board Members
4) Board Seats
5) Members
6) Admin Members
7) Agendas - (depends on the integration with http://agendas.dataomaha.com - @mitch-b)
8) Meeting Minutes

**Comment by mitch-b** [comment]
Yeah, we'll need to investigate more into what they've already done surrounding agendas. I just think it'd be great if we could tap into that versus redoing all the work they've already done. Then, we can use a third party API for agendas, and leave it out of ours. Of course we'd want to validate if we truly want it to live outside of our system or internally, since we are trying to offer a full-service solution.

Additionally, "Admin Members" could just turn into member Roles. Meeting minutes - would that be included under the Board level? Maybe we also talk about adding in document uploads. Also, how are counties/cities "registered" to start working? "Upcoming Openings" might also be bundled outside of the board schema into its own table. That way we can keep historical information.
